### PR TITLE
Style adjustments to make the top menu better

### PR DESCRIPTION
**Pull request by kevincoleman**
Fixes #

## Proposed Changes

  - Slightly taller top menu
  - Make menu items look & act like hoverable buttons instead of narrow links
  - Spacing adjustments
  - Hover effect for nav items on both light & dark mode

### CODE REVIEW

**Comment by BHydden** [comment]
Awesome!

**Comment by anoek** [comment]
I put up a forum poll about the proposed change: https://forums.online-go.com/t/feedback-requested-on-proposed-nav-bar-change/33600

**Comment by kevincoleman** [comment]
Good idea. @anoek, is there a place I can easily DM you? I have a couple questions.

**Comment by anoek** [comment]
Sure thing, anoek on online-go.com, anoek@online-go.com for email, or forums anoek

**Comment by kevincoleman** [comment]
Thanks! Just dropped you an email.

Here's one other consideration: on smaller mobile environments the menu buttons are basically useless. Should we maybe consider another change to move the items into the hamburger menu for smaller screens or something?

image

**Comment by BHydden** [comment]
They're all in the hamburger menu already, the nav bar is just redundant convenience.

**Comment by kevincoleman** [comment]
Ah! I should have checked first. Maybe we should hide them at a media query stop then, just to keep from ending up in situations like what’s above. Thoughts?

**Comment by benjaminpjones** [comment]
@kevincoleman haha I actually suggested this in the Forum post.  The cropped Home annoys me to no end (especially because all the other links are hidden *before* they get chopped)

**Comment by kevincoleman** [comment]
Well, I could probably roll that into this PR, but maybe it’s best to treat it as a different issue in case it would be received differently. I’ll see about implementing it as a beta improvement.

**Comment by kevincoleman** [comment]
Here’s an issue I just made for the items getting cropped. I would love feedback over there so we can explore options fully.

https://github.com/online-go/online-go.com/issues/1374

**Comment by anoek** [comment]
I've merged this into https://github.com/online-go/online-go.com/tree/nav-experiments , going to close this PR as it's effects will get pulled in through the `nav-experiments` branch when it's ready

**Comment by kevincoleman** [comment]
Just curious if this will ever see the light of day...

**Comment by anoek** [comment]
The "EXV6NavBar" is an opt in thing, it's flagged for moderators at the moment to work out the kinks, but I think it's ready to go, I'll unflag it so anyone can opt in.

**Comment by kevincoleman** [comment]
Ah! Thank you so much. If I make another small change is this a normal-ish timeline to expect?

**Comment by anoek** [comment]
Most small PR's are usually merged within a day or two. The issue with getting this one merged and deployed was the change altered the navigation, removing some of the links that had recently been added by community request, and reduced the board space on small devices a lot (which some folks are particularly sensitive to), so in short wasn't immediately merge-able. That being said, I did like the new style for the "normal" use case, it just needed to be paired with some more changes that rippled out into a fairly large change set, one that I didn't have the bandwidth to deal with at the time.

So, long story short, sorry it took so long, and sorry for my fail on communicating all of that at the time, I was a bit overwhelmed back then. I've flagged you as a moderator on https://beta.online-go.com/ so you can go to https://beta.online-go.com/settings/general and toggle the "enable experimental interface changes" to see it in action though.

**Comment by kevincoleman** [comment]
Oh! Thanks for the mod status, and no worries. Just trying to get a feel for what to expect.

I think maybe this got merged with some changes I didn’t intend, so it got stalled because those were more major (my PR didn’t remove any links, and only barely increased menu height in favor of tappability).

I appreciate all the work you do put in! This is a big server these days, and you do so much! Thank you. :)

**Comment by anoek** [comment]
I think your account there was kevincolemantest, but yeah if you can't get in on that one just create another and let me know so I can flag it :)

**Comment by kevincoleman** [comment]
Ahha! Thank you. I think I used credentials that matched my local dev setup (eek), so I fixed that. Appreciate it.
